docs/coding/recipes/balance-bounds.md

```diff
@@ -0,0 +1,103 @@
+---
+sidebar_position: 5
+---
+
+# Balance Bounds
+
+It is easy to limit an account's balance using either
+[`flags.debits_must_not_exceed_credits`](../../reference/account.md#flagsdebits_must_not_exceed_credits)
+or
+[`flags.credits_must_not_exceed_debits`](../../reference/account.md#flagscredits_must_not_exceed_debits).
+
+What if you want an account's balance to stay between an upper and a lower bound?
+
+This is possible to check atomically using a set of linked transfers.
+
```

Comment:
Not sure if this is worth emphasizing or if it is obvious, but:

There is a big difference between this and the "must not exceed" invariants:

With the "must not exceed" invariants, an account is guaranteed to never violate those invariants. But this maximum balance approach must be enforced per-transfer -- it is possible to exceed the limit simply by not enforcing it for a particular transfer.